### PR TITLE
Add usagebased attributes to Formbricks

### PR DESCRIPTION
**Pull request by jobenjada**
## What does this PR do?

- adds "HasThreeSurveys" and "HasTwentySubmissions" as attributes
- adds the user name as attribute

Screenshot 2023-08-15 at 17 15 29

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [X] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## Checklist

- [X] Added a screen recording or screenshots to this PR
- [ ] Filled out the "How to test" section in this PR
- [X] Read the [contributing guide](https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md)
- [X] Self-reviewed my own code
- [] Commented on my code in hard-to-understand bits
- [X] Ran `pnpm build`
- [X] Checked for warnings, there are none
- [X] Removed all `console.logs`
- [X] Merged the latest changes from main onto my branch with `git pull origin main`
- [X] My changes don't cause any responsiveness issues
- [] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/web/app/(app)/FormbricksClient.tsx [hunk 1]
```diff
@@ -16,8 +21,29 @@ export default function FormbricksClient({ session }) {
...
         formbricks.setAttribute("Plan", session.user.teams[0].plan);
+        formbricks.setAttribute("Name", session?.user?.name);
       }
     }
   }, [session]);
   return null;
 }
+
+export const updateUsageAttributes = (numSurveys, totalSubmissions) => {
+  if (!formbricksEnabled || !formbricks) return;
```
Comment (mattinannt) [inline apps/web/app/(app)/FormbricksClient.tsx:32]: `!formbricks` couldn't happen, can it?

File: apps/web/app/(app)/FormbricksClient.tsx [hunk 2]
```diff
@@ -16,8 +21,29 @@ export default function FormbricksClient({ session }) {
...
       if (session.user.teams?.length > 0) {
         formbricks.setAttribute("Plan", session.user.teams[0].plan);
+        formbricks.setAttribute("Name", session?.user?.name);
       }
     }
   }, [session]);
   return null;
 }
+
+export const updateUsageAttributes = (numSurveys, totalSubmissions) => {
```
Comment (mattinannt) [inline apps/web/app/(app)/FormbricksClient.tsx:31]: Do we need the `export`?

**Review by mattinannt — COMMENTED** [review]
@jobenjada Looks good 😊 Just left a few comments. Please check and update the code if necessary.

File: apps/web/app/(app)/FormbricksClient.tsx (hunk 1 above)
Comment (jobenjada) [reply apps/web/app/(app)/FormbricksClient.tsx]: true dat

**Pull request merged**